scripts/ci/jobs/gke_version_compatibility_tests.py

```diff
@@ -13,7 +14,7 @@
 # don't use postgres
 os.environ["ROX_POSTGRES_DATASTORE"] = "false"
 
-versions=["3.71.0", "3.70.0", "3.69.0"]
+versions=getLastSensorVersionsFromGitTagsCLI(4)
```

Comment:
I think this function name is too verbose. I suggest renaming it to `get_lastest_main_git_tags`

 1. Tags are for the main image. Not specific to sensor
 2. I think `latest` is a better word here. Because they are the _newest_ released tags.
 3. If it comes from the CLI or not is a redundant information and it doesn't really fit in the function name IMO. 
 4. You're

Reply:
Another suggestion: `get_latest_release_tags` or `get_latest_release_versions`.

Also, please don't forget to rename `sensor_versions.py` file accordingly.